packages/react-form/src/hooks/field/field.ts

```diff
@@ -198,31 +198,54 @@ export function useField<Value = string>(
   return field as Field<Value>;
 }
 
+export type ChoiceField<Value = boolean> = Omit<
+  Field<Value>,
+  'value' | 'onChange'
```

Comment:
Would users ever rely on these omitted types existing in `ChoiceField`? Would exporting a `MultiChoiceField` type make more sense?

Reply:
they shouldn't be relying on these, both are non-sensical to use in a choice component context (and `value` has never been visible)